__tests__/db.text.ts

```diff
@@ -0,0 +1,24 @@
+import dbConfig from '../src/config/dbConfig';
+
+describe('Test database connectivity', () => {
+  it('Should establish a successful database connection', async () => {
+    const { user, host, database, password, port } = dbConfig;
+    const { Pool } = require('pg');
+    const pool = new Pool({
+      user,
+      host,
+      database,
+      password,
+      port,
+    });
+
+    try {
+      await pool.query('SELECT 1');
+      console.log('Database connection successful');
+      expect(true).toBe(true);
```

Comment:
This is always true. @janithlahirukariyawasam

Reply:
Do we upload the .env file to GitHub repository?

Reply:
@janithlahirukariyawasam The `.env` should be listed on the .gitignore, to prevent the hidden variables from being committed. 
We can keep an `example.env` file. When we run the application, we can just copy the content into a `.env` file.